### DISCUSSION TITLE
20-27 Nov 2023 - Weekly Planning

### DISCUSSION THREAD

**Discussion by kjaymiller** [BlackPythonDevs/blackpythondevs#21] [Announcements]
Here's a Look at what is happening this week in the Black Python Devs Community (and around it).

## Jay Taking the second half of the week away (Jay)
 
It's Thanksgiving in the US and I'll have my family coming to visit starting as early as Tuesday. This means a lot of cleaning and cooking.

That also means that I'll be taking some time away on the discord. You may see some smaller welcomes and other communications but for the most part I will be unavailable for much outside of already scheduled events.

## Meeting with Python Podcats team (Jay)

That's not a typo. Mariatta, Georgi, and others are starting a _Hidden Figures of Python_ podcast to highlight some of the notable and diverse folks around the Python community. While I'm interested in learning 

## GitHub Projects for Productivity (Jay)

I'd like to talk about perhaps doing some level of basic project management in GitHub Projects.

Nothing so set in stone that folks have hard deadlines or anything. We just have some moving parts that I'd like to see more structure around.

I'll take the lead on building things but please feel free to make issues in .maintainers if you see room for any improvement?

## Begin conversation with designers for logo/branding

Thank you @Veldakiara for getting the contact info for the designers. I hope that we can schedule some time to chat with them in the next couple weeks.

I'm going to reach out to them by EOD Tuesday.

### Djangonauts.space testathon

https://djangonaut.space/events/2023/11/crew-commits-thanksgiving-2023/

I mentioned it would be great to do another testathon that we could promote internally and see if we could get involved.

## Everyone

You've been doing a great job with all the things.

I would love to get a list of all the things happening that we're kinda working on. Feel free to add them to the bottom of this discussion.

### @Veldakiara

Unless you have an idea for an event. Please make sure that folks know that there won't be one this week.

If you want to create an event around the [Django Testathon](#djangonauts.space-testathon) I won't stop you 😉. I would suggest reaching out to @DawnWages about it. Since it's not an internal thing I don't think we need to do much in terms of creativity. I'd love to see a good turnout from us, especially with our larger community of Django Developers

### @oleksis

I love the accessibility conversation of the site. Any chance we can get some focused content on how we're testing for accessibility on the website. It's something that I think would make an interesting blog post but would also be good for the `CONTRIBUTING.md` doc.

Also I know we were takling about using tailwind, but have you looked at some of the other class-based css frameworks. I've been playing around with Pico and looking at some of the options found [in this post](https://www.lambdatest.com/blog/best-css-frameworks/). The only one I would suggest avoiding is Bootstrap which isn't accessible by default. 

That's it for this week. Have a good one!

**Comment by oleksis** [comment]
- Happy Thanksgiving 🙏🏾🎉
- I like the idea **"Python Podcats"**
- This is great and see the big picture of the Projects **"GitHub Projects for Productivity"**
- Thanks  @VeldaKiara, "logo/branding", We need it
- This amazing, **"Djangonauts.space testathon"**, We could get involved
- I will write about "testing for accessibility" and look what CSS framework BPDevs can integrate with `Jekyll`

**Comment by VeldaKiara** [comment]
Happy thanksgiving, share some of the pie photos!!!

I will let the community know we do not have the chat this week, however I will hangout on the general voice chat to see if anyone has any questions.

I will be reaching out to Dawn in just a bit.

Enjoy!! we will be okay.

**Comment by oleksis** [comment]
- GitHub Projects for Productivity

We use a GitHub Projects for the organization (all the projects) or for each repository ?

**Reply by kjaymiller** [reply]
We have one project for all of the repos in the org

**Reply by kjaymiller** [reply]
https://github.com/orgs/BlackPythonDevs/projects/1
